shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagerSyncService.java

```diff
@@ -30,5 +30,6 @@ public void syncUpdatesFromManager() {
     public void syncStatusBackToManager() {
         LOGGER.debug("Sending back status from Operator to Manager");
         managedBridgeSyncService.syncManagedBridgeStatusBackToManager();
+        managedProcessorSyncService.syncManagedProcessorStatusBackToManager();
```

Comment:
Related to the previous comment: you could fetch all the instances from the manager, then with `onItem()` process all the instances and with another `onItem()` sync the status back to the manager. Just saying, you can do it with a single chain of `Uni`s without fetching all the instances again from the manager

Reply:
please refer to my comment above